docs/app/assets/js/search-worker.js

```diff
@@ -25,13 +26,13 @@ searchDataRequest.onload = function() {
   searchData.forEach(function(page) {
     index.add(page);
   });
-  postMessage({ e: 'index-ready' });
+  postMessage({e: 'index-ready'});
```

Comment:
Ooc, couldn't this also be `self.postMessage`? Is there any reason why you usd it below but not here (just being curious)?